services/spar/migrate-data/src/Spar/DataMigration/V1_ExternalIds.hs

```diff
@@ -186,11 +186,14 @@ sink = go
                 logDebug ("No team for user " <> show uid <> " from extid " <> show extid)
           NewExternalId (tid, extid, uid) ->
             lift $
-              dryRun <$> askMigEnv >>= \case
-                DryRun -> pure ()
-                NoDryRun ->
-                  runSpar $
-                    write insert (params LocalQuorum (tid, extid, uid))
+              askMigEnv
+                >>= ( \case
+                        DryRun -> pure ()
+                        NoDryRun ->
+                          runSpar $
+                            write insert (params LocalQuorum (tid, extid, uid))
+                    )
+                  . dryRun
```

Comment:
I strongly disagree.  what does hlint say if we put brackets around `dryRun <$> askMigEnv` in the original code?

Reply:
Yeah, I'm not sold on this either… A few suggestions seem to be counter-productive when we have code like this.

Reply:
Same, redundant `<$>`. How about extracting the case match?